app/marketing/views.py

```diff
@@ -498,3 +498,11 @@ def leaderboard(request, key=''):
         'podium_items': items[:3] if items else []
     }
     return TemplateResponse(request, 'leaderboard.html', context)
+
+
+@staff_member_required
+def day_email_campaign(request, day):
+    if day not in list(range(1, 6)):
+        raise Http404
+    response_html, _, _ = render_nth_day_email_campaign('foo@bar.com', day, "staff member")
```

Comment:
You'll want to use `response_html, __, __ =` here due to redeclaration of `gettext_lazy as _`  and avoid future confusion.